activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -29,6 +33,9 @@ def self.dump(value)
     end
 
     class InvalidMessage < StandardError; end
+    class UnexpectedAlgorithm < StandardError; end
+    class InvalidFormat < StandardError; end
```

Comment:
Are we raising all the old exceptions in the same places as we were before? Otherwise we're breaking backwards compatibility.